src/panels/lovelace/editor/config-elements/hui-entity-card-editor.ts

```diff
@@ -32,6 +32,7 @@ const cardConfigStruct = object({
   unit: optional(string()),
   theme: optional(string()),
   footer: optional(headerFooterConfigStructs),
+  severity: optional(object()),
```

Comment:
Please expand this type, it can only have the keys green, yellow red and the value should be a number.